src/Forms/Beamer.cs

```diff
@@ -288,7 +288,7 @@ private Bitmap GenerateImageFromTextWithStyle(string text)
                     sb = new StringBuilder();
 
                     int endIndex = text.Substring(i).IndexOf('>');
-                    if (endIndex == -1)

```

Comment:
I think, the assumption is, that comparing against 0 is more efficient than comparing against -1 ([here for example](https://github.com/SubtitleEdit/subtitleedit/commit/61d8bda6fc4a137152c81d1371b9e767b9aaab90#diff-65a9c06f48cd76263659510678e33a72R852)). Although [my own test](https://www.dropbox.com/s/recumj0rm8nxl6f/MinusOne-Zero.cs?dl=0) gave inconclusive results.